sklearn/covariance/_graph_lasso.py

```diff
@@ -88,112 +88,17 @@ def graphical_lasso(
     enet_tol=1e-4,
     max_iter=100,
     verbose=False,
-    return_costs=False,
     eps=np.finfo(np.float64).eps,
-    return_n_iter=False,
```

Comment:
To simplify, I make the algorithm always return all information (costs and iterations).

It avoid to have some if block in all functions.